homeassistant/components/air_quality/group.py

```diff
@@ -0,0 +1,13 @@
+"""Describe group states."""
+from typing import Callable
+
+from homeassistant.core import callback
+from homeassistant.helpers.typing import HomeAssistantType
+
+
+@callback
+def async_describe_on_off_states(
+    hass: HomeAssistantType, async_on_off_states: Callable
+) -> None:
+    """Describe group on off states."""
+    async_on_off_states(None, None)
```

Comment:
Do we still need to register something if we just pass `None`? We can just drop this file.

Reply:
Yes, None ends up in the excluded list to for https://github.com/home-assistant/core/issues/37871

Reply:
Ah it's to exclude stuff. 

I wonder if we should pass a group registration instance instead of a function.

```python
@callback
def async_describe_on_off_states(
    hass: HomeAssistantType, group_registry
) -> None:
    group_registry.exclude_domain(DOMAIN)
```

Reply:
So something like.

```
group_registry.exclude_domain()
```

```
group_registry.register_on_states([STATE_ON, OTHER_ON_STATE])
group_registry.register_off_state(STATE_OFF)
```


I guess we have to add `DOMAIN` to all of those or use a ContextVar?